rollup/proposer-tool-config.json

```diff
@@ -3,21 +3,13 @@
       "endpoint": "https://rpc.scroll.io",
       "chunk_proposer_config": {
         "max_block_num_per_chunk": 100,
-        "max_tx_num_per_chunk": 100,
         "max_l2_gas_per_chunk": 20000000,
-        "max_l1_commit_gas_per_chunk": 5000000,
-        "max_l1_commit_calldata_size_per_chunk": 123740,
         "chunk_timeout_sec": 72000000000,
-        "max_row_consumption_per_chunk": 10000000000,
-        "gas_cost_increase_multiplier": 1.2,
-        "max_uncompressed_batch_bytes_size": 634693
+        "gas_cost_increase_multiplier": 1.2
```

Comment:
Same here. Should correspond to config structs

Reply:
Yeah. Now they are removed in https://github.com/scroll-tech/scroll/commit/28fd8df526a0cda601c34f825d2b2c0c2627da4d.